test/factories/users.rb

```
@@ -0,0 +1,5 @@
+FactoryBot.define do
```

rubocop (1.69.1)

[convention] Missing frozen string literal comment.

(Style/FrozenStringLiteralComment)